docker/docker-compose.yml

```diff
@@ -19,9 +19,9 @@ services:
       # feature available in commands like docker ps.
       #
       # TODO: Remove this after startup improvement fixes
-      - ../scripts/wait-for-it.sh:/usr/local/bin/wait-for-it.sh:ro
-      - ../scripts/start-platform.sh:/usr/local/bin/start-platform.sh:ro
-      - ../scripts/start-bootstrapper.sh:/usr/local/bin/start-bootstrapper.sh:ro
+      - ../docker/scripts/wait-for-it.sh:/usr/local/bin/wait-for-it.sh:ro
```

Comment:
I guess `./scripts/` would have worked as not both are in same `docker` directory

Reply:
yes. It should work